client/src/components/App.js

```diff
@@ -160,6 +163,20 @@ var App = React.createClass({
     document.title = `#${channel}`;
     logger.debug("Set title: " + document.title);
     AppActions.setCurrentChannel(channel);
+
+    let channels = JSON.parse(localStorage.getItem( "anonet.app." + this.state.user.username + "." + this.state.networkName + ".channels")) || [];
```

Comment:
The string concatenation could be extracted into a function like `makeChannelName(username, networkName)` as it's used quite often below